platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/events/BaseEventHashedData.java

```diff
@@ -297,17 +295,17 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this, SHORT_PREFIX_STYLE)
-                .append("softwareVersion", softwareVersion)
-                .append("creatorId", creatorId)
-                .append("selfParentGen", selfParentGen)
-                .append("otherParentGen", otherParentGen)
-                .append("selfParentHash", CommonUtils.hex(valueOrNull(selfParentHash), TO_STRING_BYTE_ARRAY_LENGTH))
-                .append("otherParentHash", CommonUtils.hex(valueOrNull(otherParentHash), TO_STRING_BYTE_ARRAY_LENGTH))
-                .append("timeCreated", timeCreated)
-                .append("transactions size", transactions == null ? "null" : transactions.length)
-                .append("hash", CommonUtils.hex(valueOrNull(getHash()), TO_STRING_BYTE_ARRAY_LENGTH))
-                .toString();
+        return "BaseEventHashedData{" + "softwareVersion="
+                + softwareVersion + ", creatorId="
+                + creatorId + ", selfParentGen="
+                + CommonUtils.hex(valueOrNull(selfParentHash), TO_STRING_BYTE_ARRAY_LENGTH) + ", otherParentGen="
```

Comment:
this can be really useful information when debugging. I would say we leave it for now, events need an overhaul that I hope is coming in the near future